mollie/api/objects/customer.py

```diff
@@ -42,11 +42,9 @@ def created_at(self):
 
     @property
     def subscriptions(self):
-        url = self._get_link("subscriptions")
-
         from ..resources import CustomerSubscriptions
 
-        return CustomerSubscriptions(self.client, resource_path=url)
+        return CustomerSubscriptions(self.client, resource_path=f"customers/{self.id}/subscriptions")
```

Comment:
We could use the result from `_get_link`, but when it's empty, we still need a working instance of `CustomerSubscriptions`.